__tests__/app.test.js

```diff
@@ -94,40 +96,40 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/bananas")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('Bad request')
+        expect(body.msg).toBe("Bad request");
       });
   });
   test("Bad request- valid but out of range- returns 404 object not found", () => {
     return request(app)
       .get("/api/articles/900000")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe('Object not found')
-        });
-    });
-})
+        expect(body.msg).toBe("Object not found");
+      });
+  });
+});
 
 describe("PATCH /api/articles/:article_id", () => {
   test("takes an object and returns the updated article object", () => {
     const input = { inc_votes: 3 };
     const originArticle1 = {
-        article_id: 1,
-        title: 'Living in the shadow of a great man',
-        topic: 'mitch',
-        author: 'butter_bridge',
-        body: 'I find this existence challenging',
-        created_at: '2020-07-09T20:11:00.000Z',
-        votes: 100
-      }
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+    };
     return request(app)
       .patch("/api/articles/1")
       .send(input)
       .expect(200)
       .then(({ body }) => {
         expect(typeof body.updatedArticle).toBe("object");
-        expect(Array.isArray(body.updatedArticle)).toBe(false)
-        expect(body.updatedArticle).toBeInstanceOf(Object)
-        expect(body.updatedArticle).not.toEqual(originArticle1)
+        expect(Array.isArray(body.updatedArticle)).toBe(false);
+        expect(body.updatedArticle).toBeInstanceOf(Object);
+        expect(body.updatedArticle).not.toEqual(originArticle1);
```

Comment:
This mutation test will always pass and cannot fail, the object in the database has been changed as that is what we are testing for. The object in your test will never change as that is not what is being changed with the model's query.